### PR TITLE
UIEH-508 Move Save button to header in settings forms

### PR DESCRIPTION
**Pull request by cherewaty**
## Purpose
Other FOLIO modules have a Save button in the pane header, instead of at the bottom of a form. Completes https://issues.folio.org/browse/UIEH-508

## Approach
Put the Save button in the pane header `lastMenu` and the Cancel action in the pane header action items dropdown.

## Open Questions
- The root proxy page has some odd loading behavior (see GIF below)
- The interactors for clicking items in pane header dropdowns can be DRYed up

### CODE REVIEW

File: src/components/settings-knowledge-base/settings-knowledge-base.js [hunk 1]
```diff
@@ -31,75 +31,78 @@ class SettingsKnowledgeBase extends Component {
       invalid
     } = this.props;
 
+    let actionMenuItems = [
+      {
+        'label': 'Cancel editing',
```
Comment (elrickvm) [inline src/components/settings-knowledge-base/settings-knowledge-base.js]: Do we have to internalize these files. Or will that be in another ticket?

File: src/components/settings-knowledge-base/settings-knowledge-base.js [hunk 2]
```diff
@@ -31,75 +31,78 @@ class SettingsKnowledgeBase extends Component {
...
+              {model.update.isPending && (
+                <Icon icon="spinner-ellipsis" />
+              )}
+              <PaneHeaderButton
+                disabled={model.update.isPending || invalid || pristine}
+                type="submit"
+                buttonStyle="primary"
+                data-test-eholdings-settings-kb-save-button
+              >
+                {model.update.isPending ? 'Saving' : 'Save'}
```
Comment (elrickvm) [inline src/components/settings-knowledge-base/settings-knowledge-base.js]: i8ln?

File: src/components/settings-root-proxy/settings-root-proxy.js [hunk 3]
```diff
@@ -71,63 +71,60 @@ class SettingsRootProxy extends Component {
       });
     }
 
+    let actionMenuItems = [
+      {
+        'label': 'Cancel editing',
```
Comment (elrickvm) [inline src/components/settings-root-proxy/settings-root-proxy.js]: Same questions are 18ln?

**Comment by cherewaty** [comment]
@elrickvm yes, we should `intl` those. I reassigned the issue to you - can you take care of that tomorrow and get this merged? 😀

**Comment by elrickvm** [comment]
ok. will do!

**Pull request merged**